python/run-tests.py

```diff
@@ -158,7 +158,7 @@ def main():
     else:
         log_level = logging.INFO
     logging.basicConfig(stream=sys.stdout, level=log_level, format="%(message)s")
-    LOGGER.info("Running PySpark tests. Output is in python/%s", LOG_FILE)
+    LOGGER.info("Running PySpark tests. Output is in %s", LOG_FILE)
```

Comment:
If you see how LOG_FILE is defined, it is already a full path. 
https://github.com/tdas/spark/blob/SPARK-9572/python/run-tests.py#L69